packages/std/src/mock.rs

```diff
@@ -865,13 +866,13 @@ mod tests {
     #[test]
     fn staking_querier_validators() {
         let val1 = Validator {
-            address: Addr::unchecked("validator-one"),
+            address: String::from("validator-one"),
```

Comment:
This does a bit more than the PR title says.
I like both changes and happy to have 1 PR, maybe update title/description?

Reply:
This diff is from https://github.com/CosmWasm/cosmwasm/pull/873. I consider #873 approved now and into two separate PRs.